src/components/Logo/index.js

```diff
@@ -8,5 +10,15 @@ import useDetectMobile from '../../hooks/useDetectMobile'
 export default function Logo(){
     const {t} = useTranslation()
     const isMobile = useDetectMobile()
-    return (<img src={isMobile ? logoMobile : logo} alt={t('logo.alt')} />)
+    let logoToChoose = ''
+
+    if (isMobile) {
+        logoToChoose = logoMobile
+    } else if (window.location.pathname === '/' && !isMobile) {
+        logoToChoose = homeLogo
```

Comment:
we can use same `homeLogo.svg` for all the pages. it 's more optimized than `logo.svg`

Reply:
homeLogo.svg comes with a margin, so it is used in other pages, logo is indented to the right and the symmetry effect with "Joignez un proche en visio..." created with justify-content: space-between is lost.